src/helpers/helper.js

```diff
@@ -6,16 +6,13 @@ export const generateId = faker.random.uuid;
 export const validateInput = (input, regex) => (input ? regex.test(input) : true);
 
 export const getFavouritesList = (contactsList) => {
-  const favourites = [...contactsList];
+  const favourites = [...contactsList.filter(contact => contact.likes)];
```

Comment:
.filter being immutable, you are not required to spread the result of it in a new array. Just do:
`const favourites = contactsList.filter(contact => contact.likes);`